src/emoji.js

```diff
@@ -80,3 +77,23 @@ EMOJIBASE.forEach(emoji => {
         });
     }
 });
+
+/**
+ * Strips variation selectors from a string
+ * NB. Skin tone modifers are not variation selectors:
+ * this function does not touch them. (Should it?)
+ *
+ * @param {string} str string to strip
+ * @returns {string} stripped string
+ */
+function stripVariation(str) {
+    let ret = '';
+    for (let i = 0; i < str.length; ++i) {
+        const charCode = str.charCodeAt(i);
+        // append to output only if it's outside the variation selector range
+        if (charCode < 0xFE00 && charCode > 0xFE0F) {
+            ret += str.charAt(i);
+        }
+    }
+    return ret;
```

Comment:
can this not just be done using a regex unicode character range?

Reply:
Good question: I'm not sure as I'd have to look at whether js regexes match by UTF-16 character code or by codepoint, which I suspect depends on whether you use an old style regex or an ES6 style 'u' regex?

Reply:
nm: a regex is indeed fine. My code wasn't though because I had an 'and' when I needed an 'or'.